initfiles/componentfiles/config2xml/dali.xsd

```diff
@@ -16,6 +16,10 @@
 #    limitations under the License.
 ################################################################################
 -->
+
+
+Add a key for dali server names named dali_name that can be referred to from elsewhere that needs to select a dali.
```

Comment:
is this a todo, or a description that needs to be wrapped in XML comment tags

Reply:
XSD is current a WIP and that comment is to ensure the item is addressed. This is not production level at this time and is not included in the current set of XSDs in the newly formatted environment.